### ISSUE TITLE
profile nickname should be unlinked from username@hostname:port

### ISSUE THREAD

**Issue by cinnamonsweets** [connectbot/connectbot#633]
Profile nickname shouldn't change when username hostname or port is changed.
I can't count how many times I've made a minor change to username@hostname:port and end up having to rewrite the nickname all over again.

**Comment by stevemarple** [comment]
Yes, I've found that annoying enough I've just cloned the repository to attempt a fix. 

I think if the nickname was the same as the original `username@hostname[:port]` then it should be updated, otherwise not.

**Comment by jpopovitch** [comment]
Alternative viewpoint:  Completely changing the new nickname, when the user@hostname is modified, makes it less likely that someone can quickly (or even inadvertently) change the user@hostname without it being noticed.

**Comment by cinnamonsweets** [comment]
Like a date modified time stamp on files? It should take up a different data field that isn't user modifiable through the interface. When using send intents, you wouldn't want the nicknames changing.

**Comment by kruton** [comment]
Hmm, in the original design for this component, after a nickname was changed by the user, any change to the hostname, port, or username would not change the nickname. It seems that this is not what was implemented.

**Comment by nindanaoto** [comment]
Since #1482 was closed, we can also safely close this.
